audioqc.rb

```diff
@@ -49,8 +49,10 @@ def MediaConchScan(input)
     $policyfile.write(mcpolicy)
     $policyfile.rewind
   end
-  command = 'mediaconch --Policy=' + $policyfile.path + ' ' + '"' + input + '"'
-  mcoutcome = `#{command}`.tr('--','').tr(' ','')
+  policypath = $policyfile.path
+  command = "mediaconch --Policy=#{policypath} '#{input}'"
```

Comment:
Pretty sure the single quotes here around `#{input}` will bork this for working with windows file paths, but gonna go ahead and merge and then tweak it along with some of the other paths!

Reply:
d'oh, windows... ok thanks!

Reply:
hahaha I mean, the current way I had it of dumping to ~/Desktop makes it all moot for the moment!

Reply:
lol... shhh....